common/display/kmsfencehandler.cpp

```diff
@@ -82,6 +90,11 @@ void KMSFenceEventHandler::HandleRoutine() {
     close(kms_fence_);
     kms_fence_ = -1;
   }
+
+  for (NativeSurface *surface : surfaces_) {
+     surface->SetInUse(false);
```

Comment:
hmm after thinking bit more, I think it would be better to do this in displayqueue itself before swapping previous and present plane composition states. This will ensure we also handle surfacedamage case where we will continue to use the same overlay buffer in some cases